MariaDB/server
# MDEV-32111: Debian Sid does not have libncurses 5.x anymore

- [x] *The Jira issue number for this PR is: MDEV-32111*

## Description
Upstream Debian Sid which will become Debian Trixie (13) have dropped NCruses 5.x and changed dev package name just libncurses-dev

## How can this PR be tested?
Build branch with `debian/autobake-debs.sh` or see branch on Salsa-CI
https://salsa.debian.org/illuusio/mariadb-server/-/pipelines/592558

## Basing the PR against the correct MariaDB version

## PR quality check
